src/main/java/callofcactus/menu/LoginScreen.java

```diff
@@ -0,0 +1,196 @@
+package callofcactus.menu;
+
+import callofcactus.BackgroundRenderer;
+import callofcactus.GameInitializer;
+import callofcactus.account.Account;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.ui.TextField;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+/**
+ * @author Teun
+ */
+public class LoginScreen implements Screen {
+
+    private Vector2 usernameLabelPosition;
+    private Vector2 passwordLabelPosition;
+    private Vector2 usernameTextFieldPosition;
+    private Vector2 passwordTextFieldPosition;
+    private Vector2 loginButtonPosition;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Perhaps 'loginButtonPosition' could be replaced by a local variable.](https://www.codacy.com/app/guushamm/Call-of-Cactus/file/1388792641/issues/source?bid=2979575&fileBranchId=2980831#l29)